core/src/main/scala/com/databricks/labs/remorph/RemorphContext.scala

```diff
@@ -0,0 +1,23 @@
+package com.databricks.labs.remorph
+
+import com.databricks.labs.remorph.intermediate.{LogicalPlan, TreeNode}
+import org.antlr.v4.runtime.ParserRuleContext
+
+sealed trait RemorphContext
```

Comment:
```suggestion
sealed trait Conversion
```
what about this?

Reply:
..., software version 7.0 ...

I agree, it makes better sense